src/app/src/pages/Login.js

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { Box, Button, Input, Text, VStack } from '@chakra-ui/react';
+
+import { URLS } from '../constants';
+import { login } from '../store/authSlice';
+
+// https://stackoverflow.com/questions/39254562/csrf-with-django-reactredux-using-axios
+axios.defaults.xsrfHeaderName = 'X-CSRFTOKEN';
+axios.defaults.xsrfCookieName = 'csrftoken';
+axios.defaults.withCredentials = true;
+
+export default function Login() {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [errorDetail, setErrorDetail] = useState('');
+    const dispatch = useDispatch();
+    const navigate = useNavigate();
+
+    const loginRequest = () => {
+        axios
+            .post(URLS.LOGIN, {
+                email,
+                password,
+            })
+            .then(() => {
+                dispatch(login());
+            })
+            .catch(axiosError => {
+                setErrorDetail(axiosError.response.data?.detail);
+            });
+    };
+
+    const signedIn = useSelector(state => state.auth.signedIn);
+    const locationBeforeAuth = useSelector(
+        state => state.auth.locationBeforeAuth
+    );
+
+    useEffect(() => {
+        if (signedIn) {
+            navigate(locationBeforeAuth, { replace: true });
+        }
+    }, [navigate, signedIn, locationBeforeAuth]);
+
+    return (
+        <VStack bg='gray.100' h='100vh' spacing={5}>
+            <Box h='150px'></Box>
+            <Text textStyle='loginHeader'>Boundary Sync</Text>
+            <Box>
+                <Text textStyle='loginError'>{errorDetail}</Text>
+            </Box>
+            <Input
+                htmlSize={32}
+                width='auto'
+                bg='white'
+                aria-label='email'
+                placeholder='email'
+                onChange={({ target: { value } }) => setEmail(value)}
+                onKeyPress={e => {
+                    if (e.key === 'Enter') {
+                        loginRequest(email, password);
+                    }
+                }}
+            />
+            <Input
+                htmlSize={32}
+                width='auto'
+                bg='white'
+                type='password'
+                aria-label='password'
+                placeholder='password'
+                onChange={({ target: { value } }) => setPassword(value)}
+                onKeyPress={e => {
+                    if (e.key === 'Enter') {
+                        loginRequest(email, password);
+                    }
+                }}
+            />
+            <Button
+                w='320px'
+                variant='cta'
+                onClick={() => loginRequest(email, password)}
+            >
+                Login
+            </Button>
+            <Button variant='link'>Forgot password?</Button>
```

Comment:
@gatesgodin did we consider the possibility of underlining all text links in the app? My concern was that if we leave "Forgot password?" without underlining then we introduce a small inconsistency and a small accessibility issue, but I'm persuadable.

Reply:
I'm going to apply Terence's diff (thank you!), and we can always add underlining back later if desired.

Reply:
142ef5afd23ff83d9f8ca4a30200801e25bc31ff

Reply:
@jacobtylerwalls Hey! If we change the login page background to Grey-50 and the error message to [Red-600](https://www.figma.com/file/tJPZ0lkT5C3SAyS2g2R1aA/IoW---Boundary-Sync-Tool?node-id=235%3A6761) it will meet AAA color contrast requirements.

For the links, I'm open to underline the 'Forgot password'. There aren't many links within this app, but I do agree that they should be underlined if any surface throughout the development.

For the form labels, typically if there are 2 or less fields I opt to have the label as the placeholder for a cleaner look. Although if the form field has specific formatting (like a phone number) or many fields, then I would say that it's important to show the fields. Feel free to add them in if time permits.

Reply:
Great, I used red-600 and gray-50 in 4ccf04230ba1a462441d6db51ccf277a7dffbb2a.

I'll add the underlining back in. This is the only text link so far:
<img width="509" alt="Screen Shot 2022-09-26 at 8 19 47 AM" src="https://user-images.githubusercontent.com/38668450/192274479-a2ce7ae1-be25-4357-9880-313b36b8bf21.png">